src/cursor/abstract_cursor.ts

```diff
@@ -841,6 +871,8 @@ export abstract class AbstractCursor<
     // otherwise need to call getMore
     const batchSize = this.cursorOptions.batchSize || 1000;
     this.cursorOptions.omitMaxTimeMS = this.cursorOptions.timeoutMS != null;
+    this.cursorOptions.useMaxAwaitTimeMSAsMaxTimeMS =
```

Comment:
This will never change after the cursor is initialized.  Right?  Why re-assign it?